### DISCUSSION TITLE
Font Size?

### DISCUSSION THREAD

**Discussion by AgustinMagarinos** [IdreesInc/Monocraft#101] [Q&A]
Hi there! I'm using the font for a game using unity and for it to render properly I need the font size value. Thanks in advanced :)

**Comment by IdreesInc** [comment]
Not sure what the font size in unity is referring to, especially since fonts are made of vectors and as such can be any size. The problems you are having with rendering might be because for some reason the blocky nature of this font causes issues with certain renderers. For example, trying to use this font in Minecraft causes all sorts of weird spacing issues.
